.github/workflows/main.yml

```
@@ -0,0 +1,140 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.6', '3.7']
+
+    steps:
+    - uses: actions/checkout@v1
```

```suggestion
    - uses: actions/checkout@v2
```

It's slightly faster :smile: